packages/scss/src/components/_button.scss

```diff
@@ -56,84 +56,42 @@
 		background: transparent;
 
 		&:hover, &:focus {
-			box-shadow: 0 0 0 2px _palette("primary.color") inset;
-			color: _palette("primary.color");
+			box-shadow: 0 0 0 2px _color("primary") inset;
+			color: _color("primary");
 		}
 	}
 
 	&.mod-filling {
 		&:hover, &:focus {
-			background: _palette("primary.color");
+			background: _color("primary");
 		}
 	}
+}
 
-	/* Colors */
-
-	&.mod-primary {
-		background: _palette("primary.color");
-		color: _palette("primary.text");
-
-		&:active {
-			background: darken(_palette("primary.color"), 5%);
-		}
-	}
-
-	&.mod-secondary {
-		background: _palette("secondary.color");
-		color: _palette("secondary.text");
-
-		&:active {
-			background: darken(_palette("secondary.color"), 5%);
-		}
-	}
-
-	&.mod-grey {
-		background: _palette("grey.color");
-		color: _palette("grey.text");
-
-		&:active {
-			background: darken(_palette("grey.color"), 5%);
-		}
-	}
-
-	&.mod-light {
-		background: _palette("light.color");
-		color: _palette("light.text");
-
-		&:active {
-			background: darken(_palette("light.color"), 5%);
-		}
-	}
-
-	&.mod-error {
-		background: _palette("error.color");
-		color: _palette("error.text");
-
-		&:active {
-			background: darken(_palette("error.color"), 5%);
-		}
-	}
+// PALETTES
+// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
-	&.mod-warning {
-		background: _palette("warning.color");
-		color: _palette("warning.text");
+@mixin buttonStandardPalette($palette) {
+	background: _get($palette, "color");
+	color: _get($palette, "text");
 
-		&:active {
-			background: darken(_palette("warning.color"), 5%);
-		}
+	&:active {
+		background: darken(_get($palette, "color"), 5%);
 	}
-
-	&.mod-success {
-		background: _palette("success.color");
-		color: _palette("success.text");
-
-		&:active {
-			background: darken(_palette("success.color"), 5%);
+}
+// standard coloring
+.button:not(.mod-flat):not(.mod-wired):not(.mod-filling) {
```

Comment:
We should not have mod references outside the mod itself (or in state in some exceptions)